### PR TITLE
PDF version of the schema

### PR DESCRIPTION
**Pull request by nevrome**
For the publication and beyond I think it would be good to have a PDF version of the schema.

But I also do not want to compromise the technical setup of the schema, so I came up with this quarto document that takes exactly the files we have and renders them to PDF.

Does this look good to you?

This PR only adds the .qmd and the .pdf file. I will add another PR for a set of tiny changes I wanted to propose for the actual schema text/tables.

### CODE REVIEW

File: toPDFviaQuarto.qmd [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---
+title: Poseidon package specification \vspace{-1cm}
```
Comment (stschiff) [inline toPDFviaQuarto.qmd]: I would suggest to put the version right into the title. See for example https://samtools.github.io/hts-specs/VCFv4.2.pdf.

File: toPDFviaQuarto.qmd [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        \renewcommand{\arraystretch}{1.5}
+---
+
+<!-- can be rendered with "quarto render toPDFviaQuarto.qmd --to pdf" -->
+
+{{< include README.md >}}
+
+\newpage
+
+## Appendix
```
Comment (stschiff) [inline toPDFviaQuarto.qmd:60]: The Asterisk in some of the fields in the Appendix apparently stands for "mandatory", but it is not explained as far as I can see, within the Appendix.

**Review by stschiff — COMMENTED** [review]
OK very impressive work. The PDF is beautiful and looks very professional. Thanks! I love it, and left only very few comments.

File: toPDFviaQuarto.qmd (hunk 1 above)
Comment (nevrome) [reply toPDFviaQuarto.qmd]: Nice! I took some inspiration from that title :+1:

File: toPDFviaQuarto.qmd (hunk 2 above)
Comment (nevrome) [reply toPDFviaQuarto.qmd:60]: Good spotting. I added a sentence at the beginning of the appendix to explain that.

**Review by stschiff — APPROVED** [review]
Good to go now, from my end.

**Pull request merged**